pkgs/development/haskell-modules/patches/proto-lens-0.5.1.0.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/src/Data/ProtoLens/Encoding/Parser/Internal.hs b/src/Data/ProtoLens/Encoding/Parser/Internal.hs
```

Comment:
Is there anyway you could pull these patches from upstream?  Normally we don't like to carry around patches here in nixpkgs if at all possible to get them from upstream.

You could always just use a PR you've opened upstream as well and pull them in with `fetchPatch`.

Reply:
No. I agree with you and I tried, but nothing applied cleanly.

Reply:
Okay thanks.  In that case, let's keep it like it currently is.